docs/source/changelog.rst

```diff
@@ -28,10 +28,11 @@ Changelog
         * Add development install instructions to contributing.md (:pr:`1030`)
     * Testing Changes
         * Add ``required`` flag to CircleCI codecov upload command (:pr:`1035`)
+        * Add fixture for ``ft.demo.load_mock_customer`` (:pr:`1036`)
 
     Thanks to the following people for contributing to this release:
     :user:`frances-h`, :user:`gsheni`, :user:`kmax12`, :user:`rwedge`,
-    :user:`thehomebrewnerd`, :user:`tuethan1999`
+    :user:`thehomebrewnerd`, :user:`tuethan1999`, :user:`systemshift`
```

Comment:
Similar to the prior comment, remove your user name from here and move it up to the future release section above. This may be a little confusing, but this happened because version 0.17.0 was released as you were making your updates. These changes will be part of the next release.